src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -52,6 +53,11 @@
     private $updatePage;
 
     /**
+     * @var HistoryPageInterface
+     */
+    private $historyPage;
```

Comment:
Would be a good idea, but I think it can be implemented when we will have more things to show in history (for now there is only one page with one method).